connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/InboundDispatchingSink.java

```diff
@@ -192,19 +194,20 @@ private InboundDispatchingSink(final Connection connection,
     /**
      * Creates a Sink that dispatches inbound messages after they are mapped.
      *
-     * @param connection the connection
-     * @param headerTranslator the headerTranslator to use.
-     * @param proxyActor the actor used to send signals into the ditto cluster.
-     * @param connectionActor the connection actor acting as the grandparent of this actor.
+     * @param connection                           the connection
```

Comment:
Please undo any changes made to this file - you only reformatted the code but did not change anything.